netanalysis/traffic/data/fetch_google_traffic.py

```diff
@@ -31,14 +31,15 @@
 
 logging.getLogger().setLevel(logging.INFO)
 
+
 def main(args):
     if not args.output_dir:
         logging.error("Need to specify output directory")
         return 1
     if not os.path.exists(args.output_dir):
         os.makedirs(args.output_dir)
 
-    report = api.ApiTrafficRepository()  # type: modelTrafficRepository
+    report = api.ApiTrafficRepository()  # type: model.TrafficRepository
```

Comment:
make this type explicit if possible

report: model.TrafficRepository = api.ApiTrafficRepository()

Reply:
Done